tox.ini

```diff
@@ -3,6 +3,7 @@ skipsdist=True
 envlist=py38-linux
 tox_pip_extensions_ext_venv_update = true
 docker_compose_version = 1.26.2
+usedevelop=True
```

Comment:
i'll undo this once I have shipits as this has somehow broken CI in the past (but it speeds up local development :p)